app/views/posts/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<head>
```

Comment:
this file also does not need the `<body>` tags

You might want to also take the category out of the `<head>` and move it into an element that can be inside the body, like a `<header>` or `<div>` or `<h1>`